nb_conda_kernels/nb_conda_kernels.py

```diff
@@ -27,10 +27,10 @@ def _conda_info(self):
 
         return conda_info
 
-    def _python_executable(self):
-        """Find the python executable for each env where jupyter is installed.
+    def _all_executable(self):
```

Comment:
I agree... this is just an intermediate step to have the R working out of the box in wakari... but my plan for this later is to abstract this part to support multiples kernels...